packages/codelabs/src/basics/web-components.md

```diff
@@ -244,7 +244,7 @@ LitElement is written and distributed as es modules, this means we can import it
 
 Make sure you add `type="module"` to the script tag!
 
-Next, we need to define our class. Instead of extending `HTMLElement`, we are now extending `LitElement`. `LitElement` extends `HTMLElement` already, so we're still creating an actual custom element:
+Next, we need to define our class. Instead of extending `HTMLElement`, we are now extending `LitElement`. `LitElement` extends `HTMLElement` already, so we are still creating an actual custom element:
```

Comment:
```suggestion
Next, we need to define our class. Instead of extending `HTMLElement`, we are now extending `LitElement`. `LitElement` extends `HTMLElement` already, so we're still creating an actual custom element:
```

Reply:
I was taught to use contractions sparingly when writing to a large audience, because people that struggle with english get confused.